include/mata/nfa/nfa.hh

```diff
@@ -509,14 +504,6 @@ Nfa determinize(const Nfa& aut, std::unordered_map<StateSet, State> *subset_map
 Nfa reduce(const Nfa &aut, StateRenaming *state_renaming = nullptr,
```

Comment:
sounds like in-place thing

Reply:
We can rename it to `reduction()` or `reduced()`. Anyone else any opinions on this?